examples/parallel_for/seismic/main.cpp

```diff
@@ -53,6 +56,7 @@ RunOptions ParseCommandLine(int argc, char *argv[]) {
         utility::get_default_num_threads, 0, utility::get_default_num_threads());
 
     int numberOfFrames = 0;
+    int numberofIterations = 0;
```

Comment:
To be consistent with the `numberOfFrames`:
```suggestion
    int numberOfIterations = 0;
```